src/font.rs

```diff
@@ -655,13 +929,21 @@ pub fn parse(data: &[u8]) -> Result<Font, FontError> {
     };
     let loca = tables.get(&Tag::from_str("loca")).map(|&data| Loca(data));
     let glyf = tables.get(&Tag::from_str("glyf")).map(|&data| data);
+    let cmap = tables.get(&Tag::from_str("cmap")).map(|&data| Cmap(data));
+    let encoding_index = if let Some(ref cmap) = cmap {
```

Comment:
This can probably be rewritten simpler, something like `cmap.and_them(|cmap| cmap.find_format_4_encoding())`.